docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst

```diff
@@ -21,8 +21,8 @@ For the purpose of this guide we assume you have a storage account called `light
 We further assume the container you want to use with lightly is called `farm-animals` and already contains images.
 If you don't have a storage account or container yet follow the instructions `here <https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal>`_.
 
-Go to "Security + networking > Shared access signature". There, configure the access rights to your liking. Lightly requires at least read and list access to the container. Make sure to
-also tick the boxes for the required resource types (Container and Object). Set an expiry date and then click on "Generate SAS and connection string". Copy the generated "SAS token" including the "?"
+Go to **"Security + networking"** > **"Shared access signature"**. There, configure the access rights to your liking. Lightly requires at least read and list access to the container. Make sure to
```

Comment:
Maybe add a note that write access is required for thumbnails, videos and object level stuff.